_data/projects/dnj-colab.yml

```
@@ -0,0 +1,13 @@
+name: Dev N´ Joy Colab 
+desc: Educational Opensource Colaborative repository to improve and learn programming. Samples, Training, Projecs...and real-time coding streaming!
+site: https://github.com/smaicas-org/Dnj.Colab
+tags:
+  - c#
+  - .net
+  - opensource
+  - education
+  - community
+  - streaming
+upforgrabs:
+  name: help wanted
+  link: https://github.com/smaicas-org/Dnj.Colab/issues
```

:art: this link should be to a label in the repository:

```suggestion
  link: https://github.com/smaicas-org/Dnj.Colab/labels/help%20wanted
```